spanner/cloud-client/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -241,6 +241,36 @@ public void testSample() throws Exception {
     assertThat(out).contains("1 1 Total Junk");
     out = runSample("querywithqueryoptions");
     assertThat(out).contains("1 1 Total Junk");
+
+    out = runSample("createbackup");
+    assertThat(out).contains("Created backup");
+
+    out = runSample("cancelcreatebackup");
+    assertThat(out).contains("successfully cancelled");
+
+    out = runSample("listbackupoperations");
+    assertThat(out).contains("pending");
+
+    out = runSample("listdatabaseoperations");
+    assertThat(out).contains("optimize");
+
+    out = runSample("listbackups");
+    assertThat(out).contains(databaseId);
+
+    out = runSample("restorebackup");
+    assertThat(out).contains("Restored database [");
+
+    out = runSample("updatebackup");
+    assertThat(out).contains("Updated backup [");
```

Comment:
We ended up checking the outputs (like the backupId in listbackups) in the tests for other languages. I don't know if that's conventional in the Java tests but would it be possible to do that here?

At least in Node.js, I found it helpful to know when we hadn't defined the types correctly and it printed out "undefined".

Reply:
I've added checks for the backup ids in the logs for Java as well.